test/test/nonenative/NonenativeTests.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.nonenative;
+
+import one.profiler.test.Os;
+import one.profiler.test.Output;
+import one.profiler.test.Test;
+import one.profiler.test.TestProcess;
+
+public class NonenativeTests {
+
+    @Test(sh = "%testbin/java_first_none_native_app %s.html", output = true)
```

Comment:
What is "none native"? You mean "non-native"? There are Java applications and Native (or Non-Java) applications, but "non-native" sounds unusual.